envConfig/live.env

```diff
@@ -2,7 +2,8 @@ SIMORGH_BASE_URL=https://www.bbc.com
 SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN=https://news.files.bbci.co.uk
 SIMORGH_PUBLIC_STATIC_ASSETS_PATH=/include/articles/public/
 SIMORGH_ATI_BASE_URL=https://a1.api.bbc.co.uk/hit.xiti?
-SIMORGH_EMBEDS_BASE_URL=https://polling.bbc.co.uk
+SIMORGH_EMBEDS_BASE_URL_LIVE=https://polling.bbc.co.uk
+SIMORGH_EMBEDS_BASE_URL_TEST=https://polling.bbc.co.uk
```

Comment:
This doesn't seem right to me. A `TEST` url in the `live` env config.

Reply:
Yeah, I don't like it either, which was why I omitted it from the previous PR (which subsequently failed Cypress tests in live) :(

I am going to remove it, but I am going to add logging / fallback code in `embedUrl` to prevent the failing test but also to help try and figure out _why_